package.json

```diff
@@ -23,18 +26,18 @@
   "dependencies": {
     "bcfg": "^0.1.5",
     "bclient": "^0.1.6",
-    "bcoin": "git+https://github.com/bcoin-org/bcoin.git#a2e176d4",
-    "bcrypto": "3.0.1",
-    "bledger": "git+https://github.com/bcoin-org/bledger.git",
+    "bcoin": "github:bcoin-org/bcoin#75722469",
```

Comment:
A few months old -- you don't need a version with the correct `Bitcoin Signed Message` stuff?